customLabels_test.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"testing"
+	"github.com/stretchr/testify/assert"
+)
+
+
+
+const (
+	INCORRECT_CONSUMERGROUP_LAG_CUSTOM_LABELS = `{"consumer_notifiers": [[{"when":{ "starts_with":["string1", "string2"]}, "set": {"tags":["owner:fotocasa"]}}]}`
+	CORRECT_CONSUMERGROUP_LAG_CUSTOM_LABELS = `{
+		"consumer_notifiers": [
+			{
+				"when": {
+					"start_with": [
+						"string1",
+						"string2"
+					]
+				},
+				"set": {
+					"tags":[
+						"owner:fotocasa"
+					]
+				}
+			},
+			{
+				"when": {
+					"start_with": [
+						"string3",
+						"string4"
+					]
+				},
+				"set": {
+					"tags":[
+						"owner:mads"
+					]
+				}
+
+			}
+		]
+	}`
+)
+
+func TestNewCustomCGLagLabels_wrong_config(t *testing.T){
+	_, err := NewCustomCGLagLabels(INCORRECT_CONSUMERGROUP_LAG_CUSTOM_LABELS, 1, 1)
+
+	if err == nil {
+		t.Errorf("Json string should have failed due to the wrong string json format")
+	}
```

Comment:
We could use assert lib for this as well, https://godoc.org/github.com/realistschuckle/testify/assert#Error
```suggestion
	assert.Error(t, err, "Json string should have failed due to the wrong string json format")
```

Reply:
👍